<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AwesomeFinance</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>791</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Dashboard</string>
     </attribute>
     <widget class="QDateTimeEdit" name="dabTimeBegin">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dabTimeEnd">
      <property name="geometry">
       <rect>
        <x>613</x>
        <y>280</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
     </widget>
     <widget class="QTableView" name="KPIlist">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>421</width>
        <height>201</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>341</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="KPIcanvas"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>219</y>
        <width>421</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="dabQueryCanvas"/>
     </widget>
     <widget class="QListView" name="filterSelector">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>310</y>
        <width>341</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Bank Accounts</string>
     </attribute>
     <widget class="QTableView" name="transactionList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>421</width>
        <height>501</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListView" name="accountSelector">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>331</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>210</y>
        <width>331</width>
        <height>231</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>331</width>
         <height>177</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Add Transaction</string>
       </attribute>
       <widget class="QCheckBox" name="addChecked">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>0</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>checked</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QDateEdit" name="addDate">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="addDescription">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="addValue">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
       </widget>
       <widget class="QLineEdit" name="addCat1">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>110</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="addCat2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>110</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="addCat3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>110</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="addRun">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>130</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Running</string>
        </property>
       </widget>
       <widget class="QPushButton" name="actionAddTransaction">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>150</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Categorie</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>98</width>
         <height>28</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Filter</string>
       </attribute>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>331</width>
          <height>181</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="baTimeBegin">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="baTimeEnd">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>0</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Stock Shares</string>
     </attribute>
     <widget class="QTableView" name="stockView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QListView" name="calcModelSelector">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="getLatest">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>210</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh Stock Values</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Grouped Shares</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Credit</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Retirement</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Tax</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionLoad_xlsx"/>
    <addaction name="actionSave_Data"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuData"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_xlsx">
   <property name="text">
    <string>Load Data</string>
   </property>
  </action>
  <action name="actionSave_Data">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
